lms/djangoapps/enterprise_learner_portal/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
```

Comment:
I think we should put the `enterprise_learner_portal` app in the `edx-enterprise` repo.